ups/eupspkg.cfg.sh

```diff
@@ -28,7 +28,7 @@ if [[ $OSTYPE == darwin* ]]; then
 		# the build to pass successfully. We'll patch the resultant _galsim.so in the build() phase,
 		# enabling it to find the correct libpython*.dylib w/o the ened to set DYLD_FALLBACK_LIBRARY_PATH
 		# at runtime
-		DYLD_FALLBACK_LIBRARY_PATH=$(python -c "import sysconfig; print sysconfig.get_config_var('LIBDIR')")
+		DYLD_FALLBACK_LIBRARY_PATH=$(python -c "from __future__ import print_function; import sysconfig; print(sysconfig.get_config_var('LIBDIR'))")
```

Comment:
Again, I don't think you need `print_function`. Just add the parens in the print.